facility-manager.html

```diff
@@ -0,0 +1,791 @@
+---
+layout: default
+title: Using OMERO for Facility Managers
+menu-id: l1-workflow2
+pdfs:
+    - facility-manager.pdf
+description: Descriptions of common workflows for Facility Managers and links to useful sections of User Help and Developer documentation.
+version:
+    - yes
+---
+<div class="page-navigation">
+<h2>Contents</h2>
+<p>
+<a href="#introduction">Introduction</a><br />
+<a href="#using">Using the OMERO server</a><br />
+<a href="#permissions">Permissions on OMERO</a><br />
+<a href="#access">Access control, authentication, user accounts</a><br />
+<a href="#manage">Managing Groups</a><br />
+<a href="#import">Importing</a><br />
+<a href="#data">Data structure</a><br />
+<a href="#measure">Measurement and analysis</a><br />
+<a href="#disk">Disk usage</a><br />
+<a href="#email">Emailing users</a><br />
+<a href="#update">Updating</a><br />
+<a href="#publish">Using OMERO for publication</a>
+</p>
+</div><!-- End Page Navigation -->
+
+<div class="line-block">
+<div class="line"><br /></div>
+</div>
+
+<h2><a class="anchor" id="introduction"></a>Introduction</h2>
+
+<p>This section brings together descriptions of common workflows for Facility Managers and links to useful sections of User Help and Developer documentation.</p>
```

Comment:
Developer documentation? I'd be inclined to say 'technical documentation' given you have a link to the sysadmin docs just below this but content you're covering here isn't 'developer' content as that is categorised in the sphinx docs.